packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/diagram/services/EllipseNodeStyleProvider.java

```diff
@@ -42,7 +43,7 @@ public Optional<String> getNodeType(NodeStyleDescription nodeStyle) {
     }
 
     @Override
-    public Optional<INodeStyle> createNodeStyle(NodeStyleDescription nodeStyle, Optional<String> optionalEditingContextId) {
+    public Optional<INodeStyle> createNodeStyle(NodeStyleDescription nodeStyle, Optional<String> optionalEditingContextId, ILayoutStrategy childrenLayoutStrategy) {
```

Comment:
Wouldn't this parameter be available on nodeStyle directly? I'll have to look

Reply:
So it's a bit odd but it is used to provide a default value. I think it's not needed but we can change that another time